lib/ip.js

```diff
@@ -1,4 +1,4 @@
-var assert = require('assert');
+// var assert = require('assert');
```

Comment:
There are some commented asserts in **v4ToLong** function so that's why I decide to just comment the **assert** too. Should I remove them too ?